spec/money_spec.rb

```diff
@@ -247,11 +247,6 @@
     expect((1.50 * Money.new(1.00))).to eq(Money.new(1.50))
   end
 
-  it "is multipliable by a cents amount" do
-    expect((Money.new(1.00) * 0.50)).to eq(Money.new(0.50))
-    expect((0.50 * Money.new(1.00))).to eq(Money.new(0.50))
-  end
```

Comment:
This test is still valid (but the name of the test could be better).

Reply:
It looks to test the same as the above test? :)

Reply:
Haha, I need to learn how to scroll up 😂